src/NServiceBus.Persistence.AzureStorage.PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace NServiceBus.PersistenceTesting
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.Outbox;
+    using NServiceBus.Sagas;
+    using Persistence;
+    using Persistence.AzureStorage;
+
+    public partial class PersistenceTestsConfiguration
+    {
+        public bool SupportsDtc => false;
+
+        public bool SupportsOutbox => false;
+
+        public bool SupportsFinders => false;
+
+        public bool SupportsSubscriptions => true;
+
+        public bool SupportsTimeouts => true;
+
+        public bool SupportsPessimisticConcurrency => false;
+
+        public ISagaIdGenerator SagaIdGenerator { get; set; }
+
+        public ISagaPersister SagaStorage { get; set; }
+
+        public ISynchronizedStorage SynchronizedStorage { get; set; }
+
+        public ISynchronizedStorageAdapter SynchronizedStorageAdapter { get; set; }
+
+        public IOutboxStorage OutboxStorage { get; }
+
+        public Task Configure()
+        {
+            var connectionString = GetEnvConfiguredConnectionStringForPersistence();
+            SagaIdGenerator = new DefaultSagaIdGenerator();
+            SagaStorage = new AzureSagaPersister(connectionString, true);
+            SynchronizedStorage = new NoOpSynchronizedStorage();
+            SynchronizedStorageAdapter = new NoOpSynchronizedStorageAdapter();
+            return Task.CompletedTask;
+        }
+
+        public Task Cleanup()
+        {
+            return Task.CompletedTask;
+        }
+
+        static string GetEnvConfiguredConnectionStringForPersistence()
+        {
+            var environmentVartiableName = "AzureStoragePersistence_ConnectionString";
+            var connectionString = GetEnvironmentVariable(environmentVartiableName);
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                throw new Exception($"Oh no! We couldn't find an environment variable '{environmentVartiableName}' with Azure Storage connection string.");
+            }
+
+            return connectionString;
+        }
+
+        static string GetEnvironmentVariable(string variable)
+        {
+            var candidate = Environment.GetEnvironmentVariable(variable, EnvironmentVariableTarget.User);
```

Comment:
according to the docs https://docs.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=netcore-3.1#System_Environment_GetEnvironmentVariable_System_String_System_EnvironmentVariableTarget_ `EnvironmentVariableTarget.User` is not supported for Linux and .NET Core 3.1. Given this doesn't fail the Linux builds, I guess we can just leave it but I think this logic isn't really necessary and we can probably just use `Environment.GetEnvironmentVariable(variable)`

Reply:
Fixing this in a dedicated PR as there are more occurrences of this in this solution